internal/quirks/manufacturer/manufacturer.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package manufacturer
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/openconfig/ondatra"
+)
+
+// PortChannelName returns the port channel interface name which is
+// vendor specific.
+func PortChannelName(t testing.TB, dut *ondatra.DUTDevice, i int) string {
```

Comment:
We don't need this anymore. Ondatra now provides netutil.NextBundleInterface.

https://pkg.go.dev/github.com/openconfig/ondatra/netutil#NextBundleInterface

Example: https://github.com/openconfig/featureprofiles/blob/b8054dcbfa3c998035fd2dc8940991dd314553fc/feature/interface/aggregate/ate_tests/aggregate_test/aggregate_test.go#L432